src/coff_import_file.rs

```diff
@@ -854,79 +883,143 @@ pub fn write_import_library<W: Write + Seek>(
 
     members.push(of.create_null_thunk()?);
 
-    for e in exports {
-        if e.private {
-            continue;
-        }
-
-        let mut import_type = ImportType::Code;
-        if e.data {
-            import_type = ImportType::Data;
-        }
-        if e.constant {
-            import_type = ImportType::Const;
+    let mut add_exports = |exports: &[COFFShortExport], machine_type: MachineTypes| -> Result<()> {
+        struct Deferred<'a> {
+            name: String,
+            import_type: ImportType,
+            export: &'a COFFShortExport,
         }
+        let mut regular_imports = HashMap::new();
```

Comment:
Can you add a comment that this hashmap doesn't cause non-determinism as we never iterate over it?